presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -656,6 +663,28 @@ void PrestoServer::run() {
   }
 }
 
+void PrestoServer::startLinuxMemoryChecker() {
+#ifdef PRESTO_ENABLE_LINUX_MEMORY_CHECKER
+  uint64_t systemMemLimitBytes = systemConfig->systemMemLimitGb();
+  uint64_t systemMemShrinkBytes = systemConfig->systemMemShrinkGb();
+  linuxMemoryChecker_ =
+      std::make_unique<LinuxMemoryChecker>(PeriodicMemoryChecker::Config{
+          1'000, // memoryCheckerIntervalMs
```

Comment:
Don't think we need the comments of the Config members after each value.

Reply:
Removing